src/utility/getFormattedItemsString.js

```diff
@@ -0,0 +1,8 @@
+function getFormattedItemsString(length){
+    if(length === 1){
+        return "1 item";
+    }
+    return `${length} items`;
+}
```

Comment:
if length === 0, then returned output would be **0 items**

Condition should be if(length <= 1) {.return ${length} item} else {.....}

Reply:
for length 1 it will give 1 item else it will give items only right
for 0 and other number we will get 0items or 3items respectively